ng2-components/ng2-activiti-form/src/components/widgets/core/form-field.model.ts

```diff
@@ -340,4 +340,9 @@ export class FormFieldModel extends FormWidgetModel {
             return false;
         }
     }
+
+    getOptionName(): string {
+        let option: FormFieldOption = this.options.filter(opt => opt.id === this.value)[0];
```

Comment:
Can you also check for `length` before accessing [0], to avoid exceptions if there are no options?